util/test_base.h

```diff
@@ -41,16 +43,18 @@ class test_base {
   /** called before this test is executed
    *  @param log for emitting test notes and results
    */
-  virtual bool setup(class logger &) {
-    // stub
-    return true;
-  }
+  virtual bool setup(class logger &) { return true; }
 
-  /** execute this test
+  /** method that will be overriden in test file
    *  @param log for emitting test notes and results
    */
   virtual void run(class logger &log) = 0;
 
+  /** run overriden method with try-catch block
```

Comment:
```suggestion
  /** run overridden member function with try-catch block
```

Reply:
I've applied this comment in this two commits: 4a4aa32e58ee565190fd5c4ff444133660b1ea7b and 54f529b51c7864445834a0e62af510481bf755cf